bundles/org.eclipse.equinox.common.tests/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Automatic-Module-Name: org.eclipse.equinox.common.tests
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Bundle-ActivationPolicy: lazy
 Require-Bundle: org.junit,
- org.eclipse.equinox.common;bundle-version="3.15.100",
+ org.eclipse.equinox.common;bundle-version="3.17.000",
```

Comment:
`3.17.000` is wrong. I told you to use `3.17.0`

Reply:
Ah ok. That was my mistake. Thanks.